client/components/global-notices/index.jsx

```diff
@@ -22,15 +22,15 @@ export default function GlobalNotices( { id = 'overlay-notices' } ) {
 	const noticesList = storeNotices.map(
 		// We'll rest/spread props to notice so arbitrary props can be passed to `Notice`.
 		// Be sure to destructure any props that aren't for at `Notice`, e.g. `button`.
-		( { button, href, noticeId, onClick, onDismissClick, ...notice } ) => (
+		( { button, buttonOptions, href, noticeId, onClick, onDismissClick, ...notice } ) => (
```

Comment:
I'm adding `buttonOptions` here so we can pass them to the `NoticeAction` component

Reply:
`noticeActionProps` is more descriptive but no strong opinion.